fabbwled-backend/src/main/resources/sections/433-468.yaml

```diff
@@ -0,0 +1,132 @@
+- number: 433
+  content:
+    - text: |
+        You rush in, taking the ruffians by surprise.  
+        A couple of good buffets around the head, and they run off.  
+        The scholar lies groaning on the ground.
+    - choice:
+        - text: Help him up
+          turnTo: 172
+        - text: Rob him
+          turnTo: 379
+- number: 434
+  content:
+    - text: |
+        You can leave possessions and money here to save having to carry them around with you.
+        You can also rest here safely, and recover any Stamina points you have lost.  
+        Record in the box anything you wish to leave. Each time you return, roll two dice:
+    # TODO: Add dice functionality
```

Comment:
this fails to compile since it doesn't terminate. either remove the section or add some dummy turnTo